probe-rs/src/probe/fake_probe.rs

```diff
@@ -98,7 +98,7 @@ impl ArmProbe for &mut MockCore {
                     dhcsr |= 1 << 16;
 
                     *val = dhcsr;
-                    println!("READ  DHCSR: {:#x} = {:?}", address, val);
+                    println!("Read  DHCSR: {:#x} = {:#x}", address, val);
```

Comment:
When I run these tests, I get a bunch of println output, with no real indication of what they mean. Can we move these to logging, so that it is easier to see tests ok/fail stats?

Reply:
How are you running the tests? You shouldn't see the prints by default, if you just use `cargo test`. As log prints this is more or less useless, then you need to do additional work so that you can actually see the output.

Reply:
In VSCode, rust analyzer puts up Run/Debug affordances. I usually click on the "Run" when I'm working on individual tests or module tests.   You are correct, `cargo test` doesn't show them.